apps/storefront/src/pages/Login/index.tsx

```diff
@@ -123,12 +119,21 @@ function Login(props: PageProps) {
           await logout();
         }
 
+        if (loginFlag && companyStatusMappingKeys.includes(loginFlag)) {
+          await logout(false);
+          const i18nKey = getTranslationKeyByFlag(loginFlag);
+          if (i18nKey) {
+            snackbar.error(b3Lang(i18nKey));
+          }
+        }
         setLoading(false);
       } finally {
         setLoading(false);
       }
     })();
-  }, [b3Lang, isLoggedIn, logout, searchParams]);
+    // Ignore b3Lang and logout, as they are functions that do not depend on any reactive value
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
is this giving you trouble? disabling this eslint rule is problematic, as it can lead to serious bugs.

`b3lang` should be stable, if `useLogout` isn't, try wrapping it in a `useCallback`...